### DISCUSSION TITLE
支持 DeblurGANv2 inference

### DISCUSSION THREAD

**Discussion by vansinhu** [open-mmlab/OpenMMLabCamp#167] [MMagic]
# 支持 DeblurGANv2 inference 

发表于 ICCV 2019 的去模糊任务，开源代码链接：https://github.com/VITA-Group/DeblurGANv2

## Task Information

**Repository**：MMagic

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：xiaomile

**Claim Time**：2023年03月29日

**PR Link**：https://github.com/open-mmlab/mmagic/pull/1955

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by xiaomile** [comment]
https://github.com/open-mmlab/mmagic/pull/1955
new PR link
